daily_open_prs.rb

```diff
@@ -1,99 +1,83 @@
 #!/usr/bin/env ruby
-#This script is to output a csv file to tally how many PRs are currently open on each day between two date ranges. It is also split into both PRs raised by community members and PRs raised by Puppet members.
+# frozen_string_literal: true
+
+# This script is to output a csv file to tally how many PRs are currently open on each day between two date ranges. It is also split into both PRs raised by community members and PRs raised by Puppet members.
 
 require 'optparse'
 require 'csv'
 require 'octokit'
 require_relative 'octokit_utils'
+require 'json'
+
+output = File.read('modules.json')
+parsed = JSON.parse(output)
```

Comment:
This duplicated code can move to octokit_utils (e.g. `load_module_list`) and move below the option parsing (see comment above). No need to load the file when there is a invalid option or just `--help`.